modules/migration-configuring-proxy-for-dvm.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// * migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc
+// * migration/migrating_4_1_4/migrating-applications-with-cam-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-applications-with-cam-4-2-4.adoc
+
+[id='migration-configuring-proxy-for-dvm_{context}']
+= Configuring a proxy for direct volume migration
+
+If you are performing direct volume migration from a source cluster behind a proxy, you must configure an Stunnel proxy in the `MigrationController` custom resource (CR). Stunnel creates a transparent tunnel between the source and target clusters for the TCP connection without changing the certificates.
+
+[NOTE]
+====
+Direct volume migration supports only one proxy. The source cluster cannot access the route of the target cluster if the target cluster is also behind a proxy.
+====
+
+.Prerequisites
+
+* You must have `cluster-admin` privileges on all clusters.
+
+.Procedure
+
+. Log in to the cluster on which the `migration-controller` pod runs.
+. Get the `MigrationController` CR manifest:
++
+[source,terminal]
+----
+$ oc get migration-controller <migration_controller> -n openshift-migration
```

Comment:
Just a question. Here you use `<migration_controller>` as the name, but in the YAML below, you use `migration-controller`. Assuming they should they be consistent between the two?